src/pages/candidates/Candidates.jsx

```diff
@@ -129,6 +143,16 @@ const Candidates = () => {
 
   return (
     <Box padding="1%" width="100%" height="100%">
+       {alertMessages &&
```

Comment:
alertMessages- for errors, warnings( smth wrong).

Think anout another name.

For example, successMeesage